src/Views/SuccessView.vala

```diff
@@ -39,7 +39,7 @@ public class SuccessView : AbstractInstallerView {
         title_label.xalign = 0;
         title_label.get_style_context ().add_class ("h2");
 
-        var description_label = new Gtk.Label (_("After restarting you can set up a new user, or you can shut down now and set up a new user later."));
+        var description_label = new Gtk.Label (_("Shut down now and create a user account on the next start."));
```

Comment:
The text for `description_label` should be based on `requires_workaround()` as well. If the restart button is available, the text should say "After restarting you can set up a new user, or you can shut down now and set up a new user later". If the workaround is needed, this should say "Shut down now and create a user account on the next start".

Reply:
This should be fixed now